sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -0,0 +1,229 @@
+    record.appendWithStartFiller(this.studentAmount, 8, NUMBER_FILLER);
+    record.appendWithStartFiller(this.schoolAmount, 8, NUMBER_FILLER);
+    record.appendWithStartFiller(this.cslAwardAmount, 6, NUMBER_FILLER);
+    record.appendWithStartFiller(this.bcslAwardAmount, 6, NUMBER_FILLER);
+    record.appendDate(this.educationalStartDate, DATE_FORMAT);
+    record.appendDate(this.educationalEndDate, DATE_FORMAT);
+    record.appendWithEndFiller(this.federalInstitutionCode, 4, SPACE_FILLER);
+    record.appendWithStartFiller(this.weeksOfStudy, 2, NUMBER_FILLER);
+    record.appendWithStartFiller(this.fieldOfStudy, 2, NUMBER_FILLER);
+    record.append(this.yearOfStudy.toString(), 1);
+    record.append(this.totalYearsOfStudy.toString(), 1);
+    record.repeatAppend(SPACE_FILLER, DATE_FORMAT.length); // Cancel Date, optional, not provided.
+    record.append("F"); // 'F' for full-time. Part time is done by another integration to another system.
+    record.repeatAppend(SPACE_FILLER, 2); // Provincial field of study code, optional, not provided.
+    record.appendDate(this.enrollmentConfirmationDate, DATE_FORMAT);
+    record.append("E"); // Indicates if e-cert or paper certificate E=E-cert; P=Paper.
+    record.appendDate(this.dateOfBirth, DATE_FORMAT);
+    record.appendWithEndFiller(this.lastName, 25, SPACE_FILLER);
+    record.appendWithEndFiller(this.firstName || "", 15, SPACE_FILLER);
+    record.repeatAppend(SPACE_FILLER, 3); // Initials, optional, not provided.
+    record.appendWithEndFiller(this.addressLine1, 40, SPACE_FILLER);
+    record.appendWithEndFiller(this.addressLine2 || "", 40, SPACE_FILLER);
+    record.appendWithEndFiller(this.city, 25, SPACE_FILLER);
+    record.repeatAppend(SPACE_FILLER, 4); // Province, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 16); // Postal code, optional, not provided.
+    record.appendWithEndFiller(this.countryName, 20, SPACE_FILLER);
+    record.repeatAppend(SPACE_FILLER, 20); // Phone Number, optional, not provided.
+    record.appendWithEndFiller(this.emailAddress, 70, SPACE_FILLER);
+    record.repeatAppend(SPACE_FILLER, 40); // Alternate Address Line 1, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 40); // Alternate Address Line 2, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 25); // Alternate City, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 4); // Alternate Province, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 16); // Alternate Postal Code, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 20); // Alternate Country Name, optional, not provided.
+    record.repeatAppend(SPACE_FILLER, 20); // Alternate Phone Number, optional, not provided.
+    record.append(this.gender, 1);
+    record.append(this.maritalStatus, 1);
+    record.appendWithEndFiller(this.studentNumber || "", 12, SPACE_FILLER);
+    record.append("E"); // Student’s language preference E= English, F= French.
```

Comment:
when/where `F= French` is condition?

Reply:
@abschwenker checked on PROD and, either in the current system or this new one, we are not collecting user preferences for language. The recommendation right now is to send this always as "E" for english.

Reply:
Correct